watchman/views.py

```diff
@@ -5,26 +5,29 @@
 from django.http import Http404
 
 from jsonview.decorators import json_view
-from watchman.utils import get_checks
-from watchman.settings import WATCHMAN_CHECKS
 from watchman.decorators import token_required
+from watchman.utils import get_checks
 
 
 @token_required
 @json_view
 def status(request):
     response = {}
-    available_checks = frozenset(WATCHMAN_CHECKS)
+
+    check_list = None
+    skip_list = None
+
     if len(request.GET) > 0:
```

Comment:
It looks like this might be a bit painful to use for lists? checks / skips can be added as lists by setting them multiple times in the querystring (`?check=foo&check=bar`).

Ref: http://stackoverflow.com/questions/17159567/how-to-create-a-list-of-fields-in-django-forms

I've created https://github.com/mwarkentin/django-watchman/issues/27 to track it.